src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,19 @@
 package core.basesyntax;
 
-public class MainApp {
+import core.basesyntax.garage.Bulldozer;
+import core.basesyntax.garage.Excavator;
+import core.basesyntax.garage.Machine;
+import core.basesyntax.garage.Truck;
 
+public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine[] machines = new Machine[]{truck, bulldozer, excavator};
+        for (Machine machine : machines) {
+            machine.doWork();
+            machine.stopWork();
```

Comment:
Similarly, the message in the 'stopWork' method should also be informative. Ensure that each subclass of Machine provides a clear message indicating that the specific type of machine has stopped working.